package.json

```diff
@@ -50,8 +50,7 @@
     "freedom": "^0.6.11"
   },
   "scripts": {
-    "test": "grunt test",
-    "prepublish": "grunt test",
+    "test": "grunt build jasmine",
```

Comment:
I would expect this line to be 
```
 "test": "grunt test",
```
w.r.t. removal of `prepublish` I think that makes sense: it is not the right place for pre-submit tests and is deprecated.